docs/pages/access-controls/guides/guide-to-cluster-wide-idp-hardening.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Guide to hardening against IdP compromises
+description: Implement cluster-wide hardening measures.
+---
+
+Protecting your organization against Identity Provider (IdP) compromises is more critical than ever. 
+This guide aims to help you fortify your identity infrastructure and mitigate the risks associated with IdP weaknesses.
+
+## Understanding IdP compromise
+
+An IdP compromise occurs when an attacker gains unauthorized access to your identity management 
+system, potentially allowing them to impersonate legitimate users, escalate privileges, or access 
+sensitive information. This can happen through various means, such as exploiting software vulnerabilities, 
+stealing credentials, or social engineering attacks.
+
+## Current security measures may fall short
+
+While many organizations have implemented basic security measures like single sign-on (SSO) and multi-factor authentication (MFA), 
+these alone may not be sufficient to protect against sophisticated attacks targeting your IdP. 
+Attackers are constantly evolving their techniques, and traditional security measures may have limitations 
+or vulnerabilities that can be exploited.
+
+## Hardening measures
+
+To significantly enhance your defense against IdP compromises, we recommend implementing the following comprehensive security measures:
+
+### Set up cluster-wide WebAuthn
+Implement strong, phishing-resistant authentication across 
+your entire infrastructure using WebAuthn standards. WebAuthn, a W3C standard and part of FIDO2, 
+enables public-key cryptography for web authentication. Teleport supports WebAuthn as a second factor 
+for logging into Teleport (via tsh login or Web UI) and accessing SSH nodes or Kubernetes clusters. 
+It's compatible with hardware keys (e.g., YubiKeys, SoloKeys) and biometric authenticators like Touch ID and Windows Hello.
+
+### Establish cluster-wide per-session MFA
+Ensure that multi-factor authentication is required for each session, not just at initial login, 
+to maintain continuous security. Teleport's per-session MFA enhances security by protecting 
+against compromised on-disk certificates. It requires additional MFA checks when initiating new SSH, Kubernetes, database, or desktop sessions.
+
+### Implement cluster-wide device trust
+Develop a system to verify and manage trusted devices across your organization, reducing the risk 
```

Comment:
Please include a specific instruction on how to turn on cloud and self-hosted version here.